lib/Repository.js

```diff
@@ -491,7 +491,7 @@ class Repository extends Requestable {
     * @param {Requestable.callback} [cb] - will receive the pull request information
     * @return {Promise} - the promise for the http request
     */
-   updatePullRequst(number, options, cb) {
+   updatePullRequest(number, options, cb) {
```

Comment:
I don't think we have tests for this. This was added a few days back.

I agree this is BC but it seemed too much to need to do a bump of a major version for a typo.